var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -84,9 +104,12 @@ def build(self, spec, prefix):
         self.mech_name += self._lib_suffix
         self._build_mods('mod', link_flag, include_flag, 'mod_core')
 
-        # Store flags
-        self._incflags  = "-incflags '{}'\n".format(include_flag)
-        self._loadflags = "-loadflags '{}'\n".format(link_flag)
+        # Create rebuild script
+        with open("build_neurodamus.sh", "w") as f:
+            f.write(_REBUILD_NEURODAMUS_TPL.format(nrnivmodl=str(which('nrnivmodl')),
+                                                   incflags=include_flag,
+                                                   loadflags=link_flag))
+        os.chmod("build_neurodamus.sh", 0o770)
```

Comment:
I would have this as  `after_install` (as suggested before).

Reply:
We cant since `include_flag` etc are only avail here